cedar-policy-cli/src/lib.rs

```diff
@@ -140,23 +140,23 @@ pub struct TranslateSchemaArgs {
 /// The direction of translation
 #[derive(Debug, Clone, Copy, ValueEnum)]
 pub enum SchemaTranslationDirection {
-    /// JSON -> Human schema syntax
-    JsonToHuman,
-    /// Human schema syntax -> JSON
-    HumanToJson,
+    /// JSON -> Cedar schema syntax
+    JsonToCedar,
+    /// Cedar schema syntax -> JSON
+    CedarToJson,
 }
 
 #[derive(Debug, Clone, Copy, ValueEnum)]
 pub enum SchemaFormat {
-    /// Human-readable format
-    Human,
+    /// the Cedar format
+    Cedar,
```

Comment:
This change should be called out in the CLI changelog

Reply:
Done.